applications/DotPulse.md

```diff
@@ -0,0 +1,165 @@
+
+Dashboard Overview
+
+<img width="833" alt="DotPulse" src="https://user-images.githubusercontent.com/91743348/192284364-81506e70-8fb5-4e17-87f2-ab83fddebb20.png">
+
+### Ecosystem Fit
+
+The DotPulse project will enable all the interested parties to track the development among different Polkadot organisations on Github, to follow the number of commits, repositories, contributors, PRs, the top contributors of the month, the evolution of commits and active contributors for each month over the last year and monitor the recent commits that are being done in the various Polkadot organisations.
+
+There are many benefits of getting the DotPulse project up and running:
+ - it offers a clear overview on the open source development that is being done across the various Polkadot organizations, by having all the required data aggregated in one place
+ - acknowledges the contributors of the month
+ - displays the active contributors for each month over the last year
+ - shows the evolution of the number of commits made on GitHub by developers throughout the entire Polkadot ecosystem
+ - outlines the recent commits 
+ 
+## Team :busts_in_silhouette:
+
+### Team members
+
+  - Andreea - Team Leader
+  - Cristina - Full Stack Developer (JavaScript, React)
+  - Catalin - Full Stack Developer (JavaScript, React)
+  - Florin - UI/UX Designer (Sketch, Figma)
+
+### Contact
+
+- **Contact Name:** Andreea Stefan  
+- **Contact Email:** andreea.stefan@crosschainlabs.tech
+- **Website:** https://www.crosschainlabs.tech/
+
+### Legal Structure
+
+- **Registered Address:** Oaspetilor Boulevard, 15, Bucharest, Romania, #013865
+- **Registered Legal Entity:** CrossChain Labs SRL
+
+### Team's experience
+
+We’re CrossChain Labs, a team of of designer and software developers with hands-on experience on blockchain technology and development of decentralised applications, with previous experience as ConsenSys employees. Some of the latest dev-grants were for projects from Filecoin (https://filmarket.io/) and NEAR protocol (NEAR registrar, Audit Registry, near.link, Developer Dashboard) with tech stack: IPFS, Arweave, rust, react, go and javascript.
+We're creative, experienced, responsible, organised and do our best to make high quality work and bring ideas to life.
+
+### Team Code Repos
+
+- https://github.com/CrossChainLabs
+- https://github.com/CrossChainLabs/near-registrar
+- https://github.com/CrossChainLabs/audit-registry
+- https://github.com/CrossChainLabs/near-dns
+- https://github.com/CrossChainLabs/near-api-go
+- https://github.com/CrossChainLabs/coredns-near
+- https://github.com/CrossChainLabs/near-link
+- https://github.com/CrossChainLabs/filmarket-contract
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+
+- https://github.com/orgs/CrossChainLabs/people/yellow-cube
```

Comment:
I am unable to access any of the links provided for the Github contributors:

https://github.com/orgs/CrossChainLabs/people/yellow-cube
https://github.com/orgs/CrossChainLabs/people/aliasccl
https://github.com/orgs/CrossChainLabs/people/arctic-ash

Reply:
hey @laboon! I fixed the Github contributors' links.